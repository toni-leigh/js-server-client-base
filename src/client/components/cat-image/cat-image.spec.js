import React from 'react';
import { shallow } from 'enzyme';
import CatImage from './cat-image';

describe('<CatImage />', () => {
  let catImage;

  beforeAll(() => {
    catImage = shallow(
      <CatImage />
    );
  });

  test('basic render', () => {
    expect(catImage).toMatchSnapshot();
  });
});
