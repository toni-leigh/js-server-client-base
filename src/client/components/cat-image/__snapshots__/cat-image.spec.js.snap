// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CatImage /> basic render 1`] = `
<img
  alt="Cute kitten"
  className="crardx-cat-image"
/>
`;
